core/workflow-core/src/main/resources/storage-config.yaml

```diff
@@ -26,6 +26,23 @@ storage:
           num-retries: 10
           min-wait-ms: 100   # 0.1s
           max-wait-ms: 10000 # 10s
+  # configuration of the LakeFS & S3 for dataset storage
+  lakefs:
+    endpoint: "http://localhost:8000/api/v1"
+    auth:
+      api-secret: ""
```

Comment:
Can you add the default values that match those in docker-compose.yml? This way, we can state that the default values are provided when users install the packages using docker-compose.yml. If the services are installed manually, users can modify this file to adjust the values accordingly.

Reply:
added